packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -9,12 +14,41 @@ jsf.option({
   optionalsProbability: 1,
   fixedProbabilities: true,
   ignoreMissingRefs: true,
-  useExamplesValue: true,
-  useDefaultValue: true,
   maxItems: 20,
   maxLength: 100,
 });
 
-export async function generate(source: unknown): Promise<unknown> {
-  return jsf.resolve(cloneDeep(source));
+export function generate(source: JSONSchema): unknown {
+  return jsf.generate(cloneDeep(source));
+}
+
+export function generateStatic(source: JSONSchema): unknown {
+  const transformedSchema = toOpenAPIJSONSchemaesque(source);
+  return sampler.sample(transformedSchema);
+}
+
+function hasExamples(source: JSONSchema): source is JSONSchema6 | JSONSchema7 {
+  return 'examples' in source;
+}
+
+function toOpenAPIJSONSchemaesque(schema: JSONSchema): any {
```

Comment:
Ok, I read half of the function and it would really use some unit tests to show what it's actually doing.

Reply:
I can, but this is going to be removed soon (literally not later than Monday) — this function was just to "close" the card and make the tests pass while waiting for openapi sampler to be updated.

Reply:
https://github.com/Redocly/openapi-sampler/pull/10